### PR TITLE
Feature/cors

### PR DESCRIPTION
**Pull request by schmetzyannick**
Same problem and same solution as for: https://github.com/camunda-community-hub/zeebe-simple-monitor

Basicallythe same PR:
 -  https://github.com/camunda-community-hub/zeebe-simple-monitor/pull/477
 -  https://github.com/camunda-community-hub/zeebe-simple-monitor/pull/497
 
 Tested in same K8S setup:
 
image

### CODE REVIEW

**Comment by saig0** [comment]
@schmetzyannick thank you for contributing. :tada: 

I'll have a look at your changes soon. :eyes:

**Comment by schmetzyannick** [comment]
@saig0 did you hab any time to look at the pr?

File: README.md [hunk 1]
```diff
@@ -212,6 +213,18 @@ For example, using PostgreSQL:
 See the [docker-compose file](docker/docker-compose.yml) (profile: `postgres`) for a sample
 configuration with PostgreSQL.
 
+#### Cross Origin Requests
+
+To enable Simple Monitor to send CORS header with every HTTP response,
+add the allowed origins (`;` separated) in the following property:
```
Comment (saig0) [inline README.md:219]:
```suggestion
To enable Simple Tasklist to send CORS header with every HTTP response,
add the allowed origins (`;` separated) in the following property:
```

File: README.md [hunk 2]
```diff
@@ -212,6 +213,18 @@ For example, using PostgreSQL:
...
+
+``` 
+server:
+  allowedOriginsUrls: http://remote-hoste:8082;https://remote-hoste:8082
```
Comment (saig0) [inline README.md:223]:
```suggestion
  allowedOriginsUrls: http://localhost:8081;https://tasklist.cloud-provider.io:8081
```

**Review by saig0 — APPROVED** [review]
@schmetzyannick nice. :tada: Looks all good. :+1: 

I will adjust the README before merging. Later today, I will build a new release. :building_construction:  

Thank you for your contribution :cookie: and sorry for the delay. :bow:

**Pull request merged**